java/src/client/facade/Facade.java

```diff
@@ -283,8 +272,15 @@ public void maritimeTrade(int playerID, PortType port) throws BuildException {
      * @param playerID The ID of the player asking this
      * @return A boolean value indicating if a development card can be played
      */
-    public boolean canPlayDC(int playerID, DevCardType dc) {
-        return myTurn(playerID) && game.canPlayDevelopmentCard(playerID, dc);
+    public boolean canPlayDC(int playerID, DevCardType dc) throws PlayerExistException{
+        if(myTurn(playerID)){
+            if(dc == DevCardType.SOLDIER){return game.canUseSoldier(playerID);}
+            if(dc == DevCardType.MONUMENT){return game.canUseMonument(playerID);}
+            if(dc == DevCardType.ROAD_BUILD){return game.canUseRoadBuilder(playerID);}
+            if(dc == DevCardType.MONOPOLY){return game.canUseMonopoly(playerID);}
+            if(dc == DevCardType.YEAR_OF_PLENTY){return game.canUseYearOfPlenty(playerID);}
+        }
+        return false;
     }
```

Comment:
Should this functionality be delegated to the facade?  I'm just wondering from a design perspective, don't know the right answer.